src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -370,6 +400,7 @@ public void setYaw(Rotation2d rotation) {
      */
     public void resetOdometry(Pose2d pose) {
         // NOTE: must use pigeon yaw for odometer!
+        currentHeading = 0;
```

Comment:
Set current heading to the heading in the parameter, not 0.

Reply:
done